src/tests/ftest/pool/RebuildTests.py

```diff
@@ -398,29 +313,21 @@ def test_multipool_rebuild(self):
             if pool2.pool_info.pi_rebuild_st.rs_errno != 0:
                 self.fail("Rebuild error reported: {}"
                           .format(pool2.pool_info.pi_rebuild_st.rs_errno))
-            if pool2.pool_info.pi_rebuild_st.rs_obj_nr != objcount:
+            if pool2.pool_info.pi_rebuild_st.rs_obj_nr != self.objcount:
                 self.fail("Rebuilt objs not as expected: {0} {1}"
-                          .format(pool2.pool_info.pi_rebuild_st.rs_obj_nr,
-                                  objcount))
-            if pool2.pool_info.pi_rebuild_st.rs_rec_nr != (reccount*objcount):
+                          .format(pool2.pool_info.pi_rebuild_st.rs_obj_nr, self.objcount))
+            if pool2.pool_info.pi_rebuild_st.rs_rec_nr != (self.reccount*self.objcount):
                 self.fail("Rebuilt recs not as expected: {0} {1}".
                           format(pool2.pool_info.pi_rebuild_st.rs_rec_nr,
-                                 (reccount*objcount)))
+                                 (self.reccount*self.objcount)))
 
             # now that the rebuild finished verify the records are correct
             for tup in saved_data:
-                data2 = container2.read_an_obj(len(tup[3]), tup[1], tup[2],
-                                               tup[0], tup[4])
+                data2 = container2.read_an_obj(len(tup[3]), tup[1], tup[2], tup[0], tup[4])
```

Comment:
(pylint-line-too-long) Line too long (91/80)

Reply:
no belong to current patch.